apps/system/utils/kdbg_date.c

```diff
@@ -325,30 +325,29 @@ int kdbg_date(int argc, char **args)
 #endif
 		ret = date_settime(argc, args);
 	} else {
-#if !(defined(CONFIG_LIBC_LOCALTIME) || defined(CONFIG_TIME_EXTENDED))
```

Comment:
Why is this removed from here and moved to line 329?

Reply:
Because 5 lines from start are same between ```#if``` and ```#else```.
Those conditional affects only FORMAT. We don't need to keep same 5 lines at each case.